### ISSUE TITLE
hyparameter for InfoGAN CelebA

### ISSUE THREAD

**Issue by fjxmlzn** [conan7882/tf-gans#1]
Hi Ge,

I am trying to reproduce the InfoGAN CelebA results you show on github page. What're the hyper-parameters (learning rate, weight of InfoGAN loss, etc.) you used for that? 

Thanks!
